File: java/liskov-plus-start/src/main/java/no/bouvet/solid/lsp/external/PayPalWebService.java
package no.bouvet.solid.lsp.external;

import java.math.BigDecimal;

public class PayPalWebService {

    public String getTransactionToken(String username, String password) {
        return "Paypal token";
    }

    public String makeRefund(BigDecimal amount, String transactionId, String token) {
        return "Authorized";
    }

}

File: dotNet/LSP/LSP/Example2/External/PayPalWebService.cs
namespace Solid.LSP.Example2.External
{
    public class PayPalWebService
    {
        public string GetTransactionToken(string username, string password)
        {
            return "PayPal token";
        }

        public string MakeRefund(decimal amount, string transactionId, string token)
        {
            return "Authorized";
        }
    }
}
